Makefile

```diff
@@ -198,6 +201,20 @@ env: ## Print environment variables
 	@echo "IMG=$(IMG)"
 	@echo "CONTAINER_TOOL=$(CONTAINER_TOOL)"
 
+.PHONY: check-codespell
+check-codespell: $(CODESPELL) ## Check for spelling errors (exits with error if found)
+	@echo "🔍 Checking for spelling errors..."
+	@if $(CODESPELL) --check-filenames --skip=deploy --ignore-words=.codespell-ignore > codespell-results.txt 2>&1; then \
+		echo "✅ No spelling errors found! Cleaning up..."; \
+		rm -f codespell-results.txt; \
+		echo "🎉 Spelling check completed successfully!"; \
+	else \
+		echo "❌ Spelling errors found! Results saved to codespell-results.txt"; \
```

Comment:
IMO printing the list of spelling mistakes is enough, no need to keep a file..

Reply:
removed the saving file part.